### PR TITLE
translate create a new react app

### PR DESCRIPTION
**Pull request by galnir**

### CODE REVIEW

**Comment by itayyehezkel** [comment]
Looks Good, and seems that Netlify deployed another version.

File: content/docs/create-a-new-react-app.md [hunk 1]
```diff
@@ -1,94 +1,95 @@
...
 ```
 
->Note
+>הערה
 >
->`npx` on the first line is not a typo -- it's a [package runner tool that comes with npm 5.2+](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).
+>`npx` בשורה הראשונה זה לא שגיאת כתיב – זה [כלי הרצת חבילות שמגיע עם npm 5.2+](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).
 
-Create React App doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want. Under the hood, it uses [Babel](https://babeljs.io/) and [webpack](https://webpack.js.org/), but you don't need to know anything about them.
+Create React App לא מטפל ב- backend או במסדי נתונים, הוא רק יוצר פרונט-אנד גמיש, שניתן להשתמש בו עם כל backend שנרצה. מאחורי הקלעים, הוא משתמש ב- [Babel](https://babeljs.io/) ו- [webpack](https://webpack.js.org/), אבל אין צורך לדעת עליהם.
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
Create React App לא מטפל בלוגיקת backend או במסדי נתונים; הוא רק יוצר מערכת בניית פרונט-אנד, שניתן להשתמש בה עם כל backend שנרצה. מאחורי הקלעים, הוא משתמש ב-[Babel](https://babeljs.io/) ו-[webpack](https://webpack.js.org/), אבל אין צורך לדעת עליהם.
```

File: content/docs/create-a-new-react-app.md [hunk 2]
```diff
@@ -1,94 +1,95 @@
...
 
-When you're ready to deploy to production, running `npm run build` will create an optimized build of your app in the `build` folder. You can learn more about Create React App [from its README](https://github.com/facebookincubator/create-react-app#create-react-app--) and the [User Guide](https://facebook.github.io/create-react-app/).
+כשאתה מוכן להעלות לפרודקשן, הרצת `npm run build` ייצור גרסה יעילה של היישום שלך בתיקיית ה- `build`. ניתן ללמוד עוד על Create React App מה- [README שלו](https://github.com/facebookincubator/create-react-app#create-react-app-) ו- [מדריך המשתמש](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
כשאתה מוכן להעלות לפרודקשן, הרצת `npm run build` ייצור גרסה יעילה של היישום שלך בתיקיית ה- `build`. ניתן ללמוד עוד על Create React App מה- [README שלו](https://github.com/facebookincubator/create-react-app#create-react-app-) ו- [מדריך המשתמש](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
```

File: content/docs/create-a-new-react-app.md [hunk 3]
```diff
@@ -1,94 +1,95 @@
...
+- **[Neutrino](https://neutrinojs.org/)** משלב את הכוח של [webpack](https://webpack.js.org/) עם הפשטות של presets, ומכיל preset [ליישומי ריאקט](https://neutrinojs.org/packages/react/) [וקומפוננטות ריאקט](https://neutrinojs.org/packages/react-components/).
 
-- **[nwb](https://github.com/insin/nwb)** is particularly great for [publishing React components for npm](https://github.com/insin/nwb/blob/master/docs/guides/ReactComponents.md#developing-react-components-and-libraries-with-nwb). It [can be used](https://github.com/insin/nwb/blob/master/docs/guides/ReactApps.md#developing-react-apps-with-nwb) for creating React apps, too. 
+- **[nwb](https://github.com/insin/nwb)** מעולה במיוחד [להעלאת קומפוננטות ריאקט ל- npm](https://github.com/insin/nwb/blob/master/docs/guides/ReactComponents.md#developing-react-components-and-libraries-with-nwb). [הוא יכול לשמש](https://github.com/insin/nwb/blob/master/docs/guides/ReactApps.md#developing-react-apps-with-nwb) ליצירת יישומי ריאקט גם.
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
- **[nwb](https://github.com/insin/nwb)** מעולה במיוחד [להעלאת קומפוננטות React ל-npm](https://github.com/insin/nwb/blob/master/docs/guides/ReactComponents.md#developing-react-components-and-libraries-with-nwb). [הוא יכול לשמש](https://github.com/insin/nwb/blob/master/docs/guides/ReactApps.md#developing-react-apps-with-nwb) ליצירת אפליקציות React גם. 
```

File: content/docs/create-a-new-react-app.md [hunk 4]
```diff
@@ -1,94 +1,95 @@
...
 
-- **[nwb](https://github.com/insin/nwb)** is particularly great for [publishing React components for npm](https://github.com/insin/nwb/blob/master/docs/guides/ReactComponents.md#developing-react-components-and-libraries-with-nwb). It [can be used](https://github.com/insin/nwb/blob/master/docs/guides/ReactApps.md#developing-react-apps-with-nwb) for creating React apps, too. 
+- **[nwb](https://github.com/insin/nwb)** מעולה במיוחד [להעלאת קומפוננטות ריאקט ל- npm](https://github.com/insin/nwb/blob/master/docs/guides/ReactComponents.md#developing-react-components-and-libraries-with-nwb). [הוא יכול לשמש](https://github.com/insin/nwb/blob/master/docs/guides/ReactApps.md#developing-react-apps-with-nwb) ליצירת יישומי ריאקט גם. 
 
-- **[Parcel](https://parceljs.org/)** is a fast, zero configuration web application bundler that [works with React](https://parceljs.org/recipes.html#react).
+- **[Parcel](https://parceljs.org/)** הוא יישום מהיר, נטול קונפיגורציות שמשמש כ- bundler [ועובד עם ריאקט](https://parceljs.org/recipes.html#react).
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
- **[Parcel](https://parceljs.org/)** הוא אפליקציית ווב מהירה, נטולת קונפיגורציות שמשמשת כ-bundler [שעובד עם React](https://parceljs.org/recipes.html#react).
```

File: content/docs/create-a-new-react-app.md [hunk 5]
```diff
@@ -1,94 +1,95 @@
...
 
-- **[Razzle](https://github.com/jaredpalmer/razzle)** is a server-rendering framework that doesn't require any configuration, but offers more flexibility than Next.js.
+- **[Razzle](https://github.com/jaredpalmer/razzle)** הוא פרייוורק שמשמש ל- server-rendering ולא דורש קונפיגורציה, אך מציע יותר גמישות מ- Next.js.
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
- **[Razzle](https://github.com/jaredpalmer/razzle)** הוא פרייוורק שמשמש לרינדור צד-שרת ולא דורש שום קונפיגורציה, אך מציע יותר גמישות מ-Next.js.
```

File: content/docs/create-a-new-react-app.md [hunk 6]
```diff
@@ -1,94 +1,95 @@
...
 
-## Creating a Toolchain from Scratch {#creating-a-toolchain-from-scratch}
+## יצירת כלי מאפס {#creating-a-toolchain-from-scratch}
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
## יצירת סט כלים מאפס {#creating-a-toolchain-from-scratch}
```

File: content/docs/create-a-new-react-app.md [hunk 7]
```diff
@@ -1,94 +1,95 @@
...
 
-A JavaScript build toolchain typically consists of:
+בדרך כלל כלי ג'אווהסקריפט מכיל:
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
בדרך כלל סט כלי JavaScript מורכב מ:
```

File: content/docs/create-a-new-react-app.md [hunk 8]
```diff
@@ -1,94 +1,95 @@
...
 
-* A **package manager**, such as [Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/). It lets you take advantage of a vast ecosystem of third-party packages, and easily install or update them.
+* **מנהל חבילות**, כמו [Yarn](https://yarnpkg.com/) או [npm](https://www.npmjs.com/). נותן לך אפשרות להתקין או לעדכן בקלות חבילות צד שלישי.
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
* **מנהל חבילות**, כמו [Yarn](https://yarnpkg.com/) או [npm](https://www.npmjs.com/). זה מאפשר לך לנצל אקוסיסטם רחב של חבילות צד-שלישי, ולהתקין או לעדכן אותן בקלות.
```

File: content/docs/create-a-new-react-app.md [hunk 9]
```diff
@@ -1,94 +1,95 @@
...
 
-* A **bundler**, such as [webpack](https://webpack.js.org/) or [Parcel](https://parceljs.org/). It lets you write modular code and bundle it together into small packages to optimize load time.
+* **bundler**, כמו [webpack](https://webpack.js.org/) או [Parcel](https://parceljs.org/). נותן לך לכתוב קוד מודולרי ולקבץ אותו לחבילות קטנות על מנת לשפר זמני טעינה.
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
* **bundler**, כמו [webpack](https://webpack.js.org/) או [Parcel](https://parceljs.org/). זה מאפשר לך לכתוב קוד מודולרי ולקבץ אותו לחבילות קטנות על מנת לשפר זמני טעינה.
```

File: content/docs/create-a-new-react-app.md [hunk 10]
```diff
@@ -1,94 +1,95 @@
...
 
-* A **compiler** such as [Babel](https://babeljs.io/). It lets you write modern JavaScript code that still works in older browsers.
+* **compiler** כמו [Babel](https://babeljs.io/). נותן לך לכתוב קוד ג'אווהסקריפט מודרני שעדיין עובד בדפדפנים ישנים.
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
* **compiler** כמו [Babel](https://babeljs.io/). זה מאפשר לך לכתוב קוד JavaScript מודרני שעדיין עובד בדפדפנים ישנים.
```

File: content/docs/create-a-new-react-app.md [hunk 11]
```diff
@@ -1,94 +1,95 @@
...
 
-If you prefer to set up your own JavaScript toolchain from scratch, [check out this guide](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) that re-creates some of the Create React App functionality.
+אם אתה מעדיף לתכנן את הכלי שלך מאפס, [קרא את המדריך הזה](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) שיוצר מחדש כמה מהפונקציונליות של Create React App.
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
אם אתה מעדיף לתכנן את סט הכלים שלך מאפס, [קרא את המדריך הזה](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) שיוצר מחדש כמה מהפונקציונליות של Create React App.
```

File: content/docs/create-a-new-react-app.md [hunk 12]
```diff
@@ -1,94 +1,95 @@
...
 
-Don't forget to ensure your custom toolchain [is correctly set up for production](/docs/optimizing-performance.html#use-the-production-build).
+אל תשכח לבדוק שהכלי שלך [מתוכנן באופן נכון לפרודקשן](/docs/optimizing-performance.html#use-the-production-build).
```
Comment (galtalmor) [inline content/docs/create-a-new-react-app.md]:
```suggestion
אל תשכח לוודא שסט הכלים שלך [מוגדר באופן נכון לפרודקשן](/docs/optimizing-performance.html#use-the-production-build).
```

**Comment by galnir** [comment]
Thanks @galtalmor 👍

**Pull request merged**